File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/DomainBaseRepo.java
package com.platdmit.simplecloudmanager.domain.repo;

import com.platdmit.simplecloudmanager.domain.models.Domain;

import java.util.List;

import io.reactivex.rxjava3.core.Observable;

public interface DomainBaseRepo {
    Observable<List<Domain>> getDomains();
    Observable<Domain> getDomain(long id);
    void nextUpdate();
}

File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/DomainBaseRepo.kt
package com.platdmit.simplecloudmanager.domain.repo

import com.platdmit.simplecloudmanager.domain.models.Domain
import io.reactivex.rxjava3.core.Observable

interface DomainBaseRepo {
    fun getDomains(): Observable<List<Domain>>
    fun getDomain(id: Long): Observable<Domain>
    fun nextUpdate()
}